src/Microsoft.AspNetCore.Routing/project.json

```diff
@@ -1,5 +1,9 @@
 {
-  "description": "ASP.NET 5 middleware for routing requests to application logic and for generating links.",
+  "description": "ASP.NET Core middleware for routing requests to application logic and for generating links.\r\nCommonly used types:\r\n Microsoft.AspNetCore.Routing.Route\r\nMicrosoft.AspNetCore.Routing.RouteCollection",
```

Comment:
Not a huge fan of it either, but it *is* correct, and I can't think of anything better :smile: